support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -694,6 +706,14 @@ function CheckoutComponent({ geoId }: Props) {
 		}
 	};
 
+	function getLabelText(label: string, checked: boolean): JSX.Element {
+		return (
+			<p css={checked ? checkedRadioLabelColour : defaultRadioLabelColour}>
```

Comment:
Is there a reason we use a `p`? I think this get's wrapped in a `label` in the `Radio` component further down, which feels like odd semantics.

Reply:
No reason, have removed the `p` now, and styled the `label div` directly. Updated.